charts/clearblade-iot-enterprise/CHANGELOG.md

```diff
@@ -40,4 +40,8 @@ ClearBlade: Adds helm value for database config toml
 
 ClearBlade: Add platform flag config for messagingURL
 
-Haproxy: Add new optional port for secondary unsecure mqtt traffic 
\ No newline at end of file
+Haproxy: Add new optional port for secondary unsecure mqtt traffic 
+
+## [3.4.0] - 2025-09-17
```

Comment:
should add note about service account perms change. we handle the secret format conversion in the controller itself, right? so no need for a note about manual conversion?

Reply:
Added it. Yes, secret conversion is automatic, no reason for the user to be aware of the change.